### ISSUE TITLE
Spaces in links are displayed are "%20"

### ISSUE THREAD

**Issue by ijreilly** [twentyhq/twenty#6381]
https://github.com/user-attachments/assets/c2fac116-269e-47b0-8179-129e12f7da91

**Comment by Faisal-imtiyaz123** [comment]
@ijreilly It is working fine for me.

**Comment by ehconitin** [comment]
@ijreilly How do I reproduce this bug? because it works fine for me too.

**Comment by ehconitin** [comment]
Hi @ijreilly,

I looked into the issue, and it appears to be caused by how URL encoding happens in the `LinksDisplay.tsx` [Code](https://github.com/twentyhq/twenty/blob/main/packages/twenty-front/src/modules/ui/field/display/components/LinksDisplay.tsx#L51) compoenent where `getUrlHostName` utility function ([LINK](https://github.com/twentyhq/twenty/blob/main/packages/twenty-front/src/utils/url/getUrlHostName.ts#L6)).

Cause of the Issue
When using the URL constructor to parse a URL, any special characters, including spaces, are automatically percent-encoded. This means spaces in the URL are converted to %20, which can lead to unexpected results in some cases.

So i dug further, it turns out there is another way used to fetch those `absoluteURLs` , `labels` and `type` which is used in `LinkDisplay.tsx`[Code](https://github.com/twentyhq/twenty/blob/main/packages/twenty-front/src/modules/ui/field/display/components/LinkDisplay.tsx#L21)

So i tried using the logic from `LinksDisplay.tsx` and it works.
If this logic is shared I think there should be a separate util for this, but there might be a reason on why was util `getUrlHostName` introduced but it is not working for this particular use case.

let me know what should I go with, should I just use the same code twice or create a new util for this?

Fix:
In this fix the url does have %20 in the place of spaces but the labels does not. Which is what we exactly want.

https://github.com/user-attachments/assets/c07d5fce-664b-44fb-947a-dfdc71e2941c

**Comment by ijreilly** [comment]
Hi @ehconitin, thanks a lot for the time you spent on this issue and your detailed analysis. 
I looked closely at your code, the approach is good!, but after a discussion with my coworker @lucasbordeau we feel this will impact performance too much if we start parsing data at each render for each cell. Performance is a key issue for us on the table views as you can imagine. 
Actually the issue is probably that you should not be able to input spaces in the url (otherwise it's no longer a url). Maybe something to work on later; in the meantime I will close this issue. I hope you understand. thanks again!

**Comment by ehconitin** [comment]
@ijreilly,

I agree that performance should be our key priority. I also concur that URLs should not contain spaces.

Thank you for your reply.
